### PR TITLE
Cleaned up Tensorflow API again.

### PR DESCRIPTION
**Pull request by kentonl**
Just over 3000 sentences/second by getting rid of the the softmax layer :)

Also removes some unused code from the CNN supertagger to avoid confusion.

### CODE REVIEW

**Comment by kentonl** [comment]
Also, the code for generating the external jars and binaries is now publicly available at this fork of TensorFlow:
https://github.com/kentonl/tensorflow/tree/taggerflow

**Comment by mikelewis0** [comment]
Looks much cleaner, thanks! And 3000 sentences per second is awesome...
